web/src/components/layout/Loading.tsx

```diff
@@ -31,11 +31,37 @@ const Layout = (props: LayoutProps) => (
   <PlainLayout mountHeader={false} mountSidebar={false} {...props} />
 );
 
+type LoadingProps = {
+  /** Text to be rendered alongside the spinner */
+  text?: string;
+  /**
+   * Whether the loading screen should listen for and render any questions
```

Comment:
NP: listen for or listen to?

Reply:
You know English isn't really my thing ;) But from what I understand, in this context it’s better to use _listen for_

* https://www.collinsdictionary.com/dictionary/english/listen (entries 2 and 3)
* https://ell.stackexchange.com/questions/118448/listen-to-or-listen-for
* https://hinative.com/questions/4216094